src/python/WMCore/ReqMgr/Utils/Validation.py

```diff
@@ -81,6 +81,10 @@ def validate_request_create_args(request_args, config, reqmgr_db_service, *args,
     3. convert data from body to arguments (spec instance, argument with default setting)
     TODO: raise right kind of error with clear message
     """
+    ### TODO: backwards compatibility. Remove these 2 lines in ~HG1805
+    request_args.pop('MaxRSS', None)
+    request_args.pop('MaxVSize', None)
+
```

Comment:
nice. :-)

Reply:
And I'm proud of it LoL . Actually we need it for ACDCs and clones only.

Reply:
Don't we need before assignment as well. (If wf is created before the update?) Anyway, you cover both cases.

Reply:
Nope, otherwise Ops will never update their scripts :)
If wf is created after HG1711 release, then it should have the proper Memory/MaxRSS settings.